verifier/src/report_body.rs

```diff
@@ -24,16 +24,35 @@ pub trait Accessor<T>: Debug {
     fn get(&self) -> T;
 }
 
-impl Accessor<Attributes> for ReportBody {
-    fn get(&self) -> Attributes {
-        self.attributes()
-    }
+/// Macro to generate boilerplate for implementing [`Accessor`] for a member of
+/// [`ReportBody`].
+///
+/// Will create [`Accessor`] implementations for both `ReportBody` and the
+/// underlying member type
+///
+/// # Arguments
+/// * `member_type` - The type of the member
```

Comment:
nit: Can you clarify what `member` refers to here?

Reply:
I took a stab at this, but I'm up for suggestions on more clarity.
I renamed to  "field" as that seemed to be used more often in rust lingo.
It feels a little awkward as none of the fields of `ReportBody` are public so they're only accessible through methods on `ReportBody`